packages/rum/test/mockWorker.ts

```diff
@@ -64,41 +65,47 @@ export class MockWorker implements DeflateWorker {
       switch (message.action) {
         case 'init':
           this.listeners.message.forEach((listener) =>
-            listener({
-              data: {
-                type: 'initialized',
-              },
-            })
+            listener(
+              createNewEvent('message', {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Do we need to use real events in the mock worker? types allow us to use simple structures

Reply:
True. I felt using `createNewEvent` to centralize the `event.__ddIsTrusted = true` quite handy. Do you see an issue in having real events?

Reply:
Not an issue, but it is a bit unnecessary (also `__ddIsTrusted` isn't used here)

Reply:
Well, FMU it is used because of: https://github.com/DataDog/browser-sdk/blob/53ed383dbfd28ece437db0e813d065d539ee045d/packages/rum/src/domain/segmentCollection/segment.ts#L44-L46

Reply:
Aah right nevermind!